Katas UI strings review - The Qubit (#1512)
microsoft/qsharp: katas/content/qubit/index.md

\alpha \cdot \begin{bmatrix} 1 \\ 0 \end{bmatrix} + \beta \cdot \begin{bmatrix} 0 \\ 1 \end{bmatrix}
$$

Because of this, qubit states $0$ and $1$ are known as **basis states**. These two vectors have two properties.

1. They are normalized.

    $$
    \langle \begin{bmatrix} 1 \\ 0 \end{bmatrix} , \begin{bmatrix} 1 \\ 0 \end{bmatrix} \rangle =
    \langle \begin{bmatrix} 0 \\ 1 \end{bmatrix} , \begin{bmatrix} 0 \\ 1 \end{bmatrix} \rangle = 1
    $$

> 1. They are orthogonal to each other.

Reviewer: Do we have to use the same number for list items? It reads easier for the person working with Markdown if they're different

    </tr>
</table>

The kets $\ket{0}$ and $\ket{1}$ represent basis states, so they can be used to represent any other state:

$$\begin{bmatrix} \alpha \\ \beta \end{bmatrix} = \alpha\ket{0} + \beta\ket{1}$$

Dirac notation isn't restricted to vectors $0$ and $1$; it can be used to represent any vector, similar to how variable names are used in algebra. For example, you can call the above state "$\psi$" and write it as:

$$\ket{\psi} = \alpha\ket{0} + \beta\ket{1}$$

> Several ket symbols have a generally accepted use, so you will see them often. For example, the following kets are used to represent the states that are superpositions of the basis states:

Reviewer: This addition looks odd, since most kets are used to represent the states that are superpositions of the basis states... How about
```suggestion
Several ket symbols have a generally accepted use, so you will see them often. For example, the following kets are commonly used:
```

In Q#, qubits are represented by the `Qubit` data type. On a physical quantum computer, it's impossible to directly access the state of a qubit, whether to read its exact state, or to set it to a desired state, and this data type reflects that. Instead, you can change the state of a qubit using quantum gates, and extract information about the state of the system using measurements.

> Thus, when you run Q# code on a quantum simulator instead of a physical quantum computer, you can use diagnostic functions that allow you to peek at the state of the quantum system. This is very useful both for learning and for debugging small Q# programs.

Reviewer: This change changes meaning: "thus" means "as a consequence of this", and the previous paragraph says that you cannot access the state. Here, "that being said" disagrees with the previous paragraph and points out an exception. Please revert or rephrase
